blazor/gantt-chart/split-task.md

```diff
@@ -0,0 +1,247 @@
+---
+layout: post
+title: Split tasks in Blazor Gantt Chart Component | Syncfusion
+description: Checkout and learn here all about split tasks in Syncfusion Blazor Gantt Chart component and much more.
+platform: Blazor
+control: Gantt Chart
+documentation: ug
+---
+
+# Split Tasks in Blazor Gantt Chart Component
+
+In [Blazor Gantt Chart](https://www.syncfusion.com/blazor-components/blazor-gantt-chart), The split taskbar feature empowers users to divide taskbars seamlessly whenever idle days occur in our project. This functionality not only facilitates a dynamic and interactive view but also enhances the visualization of the project taskbar. The split task support also working both project and resource view.
+
+## Split Tasks collection
```

Comment:
Titles must in sentence casing. First word only be starts with cap letter rest of the words be in small letter. Do the correction for all the topics